index_zh-cn.html

```diff
@@ -10,6 +10,7 @@
   createpackages: 轻松创建您的 Homebrew 程式。
   hack: 以 git、 ruby 为其筋骨，所以借助您的相关知识，自由修改，并且可以简单撤回您的调改或者合并上游更新。
   formula: Homebrew 的程式都是简单的 Ruby 脚本：
+  editor: 使用 $EDITOR 編輯!
```

Comment:
@xu-cheng Fixed. To be honest, I had to zoom in my editor to understand how these are different.